linux/README.md

```diff
@@ -53,6 +53,12 @@ Example:
     -PJDK_ARCHITECTURE=x64
 ```
 
+#### Optional Arguments
```

Comment:
Ideally, this section should explain how third parties can make packages with their own naming scheme. It should also mention that company names are supposed to go into the version.